virtual/sound-theme/sound-theme-0.ebuild

```diff
@@ -0,0 +1,18 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Virtual for themes following the freedesktop.org sound naming spec"
+
+SLOT="0"
+KEYWORDS="~alpha amd64 arm arm64 ~hppa ~loong ~mips ppc ppc64 ~riscv ~s390 sparc x86 ~amd64-linux ~x86-linux"
+IUSE="+sound"
```

Comment:
Why does it have a USE at all?

Also, new virtuals have to go to the ML.

Reply:
Because it was RDEPEND optional in libcanberra before, and this was just moved from there.

Reply:
See also: https://bugs.gentoo.org/941986#c2

@pacho2 ^